tests/inspect.bats

```diff
@@ -2,6 +2,16 @@
 
 load helpers
 
+@test "inspect" {
+	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+	run buildah commit --signature-policy ${TESTSDIR}/policy.json "$cid" alpine-image
+	[ "$status" -eq "0" ]
+	run buildah inspect alpine
```

Comment:
Would it be possible to check some fields in these images to make sure they are the same.
Would probably make for  a better tests.

Check Cmd, Entrypoint, Env, Labels ...

Reply:
updated, WDYT?